core/build.sbt

```diff
@@ -3,8 +3,17 @@ libraryDependencies += "org.yaml" % "snakeyaml" % "1.26"
 addSbtPlugin("io.crashbox"       % "sbt-gpg"            % "0.2.1")
 addSbtPlugin("com.codecommit"    % "sbt-github-actions" % "0.1-58d9629")
 addSbtPlugin("de.heikoseeberger" % "sbt-header"         % "5.4.0")
-addSbtPlugin("com.dcsobral"      % "sbt-trickle"        % "0.2.4")
+addSbtPlugin("com.dcsobral"      % "sbt-trickle"        % "0.3-37c1a2c")
```

Comment:
I'm waiting to be certain of what the API needs to look like before I make a release.